src/helm/config/model_deployments.yaml

```diff
@@ -1036,6 +1036,25 @@ model_deployments:
     client_spec:
       class_name: "helm.clients.huggingface_client.HuggingFaceClient"
 
+  # Alibaba DAMO Academy
+  - name: huggingface/seallm-7b-v2
+    model_name: damo/seallm-7b-v2
+    tokenizer_name: damo/seallm-7b-v2
+    max_sequence_length: 4096
+    client_spec:
+      class_name: "helm.clients.huggingface_client.HuggingFaceClient"
+      args:
+        pretrained_model_name_or_path: SeaLLMs/SeaLLM-7B-v2
+
+  - name: huggingface/seallm-7b-v2.5
+    model_name: damo/seallm-7b-v2.5
+    tokenizer_name: google/gemma-7b
```

Comment:
Is this the best tokenizer for this model, the docs seem to indicate they want us to use theirs like in v2 above

https://huggingface.co/SeaLLMs/SeaLLM-7B-v2.5

Reply:
Based on file size, they look the same, but I'll switch to using their tokenizer just in case.